app/templates/partials/footer.html

```diff
@@ -0,0 +1,58 @@
+<footer class="footer" role="contentinfo">
+    <div class="container">
+        <div class="grid">
+            <div class="grid__col col-4@m">
+                <p class="venus footer__heading"><strong>Legal information</strong></p>
+                <ul class="list list--bare">
+                    <li>
+                        <a href="/cookies-privacy"
+                           class="footer__link">Cookies and privacy</a>
+                    </li>
+                </ul>
+            </div>
+
+            <div class="grid__col col-4@m">
+                <p class="venus footer__heading"><strong>About ONS</strong></p>
+                <ul class="list list--bare">
+                    <li>
+                        <a href="https://www.ons.gov.uk/surveys/informationforhouseholdsandindividuals/householdandindividualsurveys"
+                           class="footer__link">What we do</a>
+                    </li>
+                    <li>
+                        <a href="/contact-us"
```

Comment:
Likewise, there is no `/contact-us` path. "https://www.ons.gov.uk/aboutus/contactus"?